frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/MembersPagePage/MembersPage.tsx

```diff
@@ -0,0 +1,231 @@
+  );
+
+  const onRemoveMemberSubmit = useCallback(() => {
+    const removedUsers = { users: [{ id: userEditing?.id }] };
+    dispatch(teamActions.removeMembers(teamId, removedUsers))
+      .then(() => {
+        dispatch(
+          renderFlash("success", `Successfully removed ${userEditing?.name}`)
+        );
+      })
+      .catch(() => console.log("error removing"));
+    toggleRemoveMemberModal();
+  }, [
+    dispatch,
+    teamId,
+    userEditing?.id,
+    userEditing?.name,
+    toggleRemoveMemberModal,
+  ]);
+
+  const onAddMemberSubmit = useCallback(
+    (newMembers: INewMembersBody) => {
+      dispatch(teamActions.addMembers(teamId, newMembers)).then(() => {
+        dispatch(
+          renderFlash(
+            "success",
+            `${newMembers.users.length} members successfully added to TEAM.`
+          )
+        ); // TODO: update team name
+      });
+      toggleAddUserModal();
+    },
+    [dispatch, teamId, toggleAddUserModal]
+  );
+
+  const onEditMemberSubmit = useCallback(
+    (formData: IFormData) => {
+      const updatedAttrs = userManagementHelpers.generateUpdateData(
+        userEditing as IUser,
+        formData
+      );
+
+      dispatch(userActions.update(userEditing, updatedAttrs))
+        .then(() => {
+          dispatch(renderFlash("success", "User updated"));
+        })
+        .catch(() => {
+          dispatch(renderFlash("error", "User failed"));
+        });
+      toggleEditMemberModal();
+    },
+    [dispatch, toggleEditMemberModal, userEditing]
+  );
+
+  // NOTE: this will fire on initial render, so we use this to get the list of
+  // users for this team, as well as use it as a handler when the table query
+  // changes.
+  const onQueryChange = useCallback(
+    (queryData) => {
+      const { pageIndex, pageSize, searchQuery } = queryData;
+      dispatch(
+        userActions.loadAll({
+          page: pageIndex,
+          perPage: pageSize,
+          globalFilter: searchQuery,
+          teamId,
+        })
+      );
+    },
+    [dispatch, teamId]
+  );
+
+  const onActionSelection = (action: string, user: IUser): void => {
+    switch (action) {
+      case "edit":
+        toggleEditMemberModal(user);
+        break;
+      case "remove":
+        toggleRemoveMemberModal(user);
+        break;
+      default:
+    }
+  };
+
+  const tableHeaders = generateTableHeaders(onActionSelection);
+
+  const loadingTableData = useSelector(
+    (state: IRootState) => state.entities.users.loading
+  );
+  const users = useSelector((state: IRootState) =>
+    generateDataSet(teamId, state.entities.users.data)
+  );
+  const team = useSelector((state: IRootState) => {
+    return state.entities.teams.data[teamId];
+  });
+  const teams = useSelector((state: IRootState) => {
+    return memoizedGetTeams(state.entities.teams.data);
+  });
+
+  return (
+    <div className={baseClass}>
+      <p>Add, customize, and remove members from Walmart Pay.</p>
```

Comment:
This should use the team name from the team object

Reply:
ah yes sorry just missed that one. will add